.sauce/playwright.yml

```diff
@@ -1,47 +1,44 @@
 apiVersion: v1alpha
 kind: playwright
-defaults:
-  mode: sauce
 sauce:
   region: us-west-1
-  concurrency: 1
+  concurrency: 10
   metadata:
     name: Testing Playwright Support
     tags:
       - e2e
     build: "$BUILD_ID"
 docker:
   fileTransfer: mount
+rootDir: tests/e2e/playwright/
 playwright:
-  projectPath: tests/e2e/playwright/
-  version: 1.7.1
-  params:
-    browserName: firefox
+  version: 1.10.0
```

Comment:
latest version of playwright is `1.11.1`. It'd be better to update to that version.